puphpet/config.yaml.template

```diff
@@ -208,13 +264,20 @@ python:
 mysql:
     install: '1'
     override_options: {  }
-    root_password: '123'
+    root_password: password
```

Comment:
Quotes are necessary if you need to handle character escape sequences in the value. Single quotes and double-quotes are treated differently. puphpet.com's generator itself doesn't seem to be consistent on usage.